server/db/seeds.rb

```diff
@@ -0,0 +1,9 @@
+ApiUser.destroy_all
+
+ApiUser.create([
+                 {first_name: 'Mary', last_name: 'Black', city: 'Dublin', mentor: true, mentee: false, email: 'Test_email_1@email.com', password_digest: 'password1'}, 
+                 {first_name: 'Bob', last_name: 'Dylan', city: 'New York', mentor: true, mentee: false, email: 'Test_email_2@email.com', password_digest: 'password2'}, 
+                 {first_name: 'Jane', last_name: 'Smith', city: 'Edinburgh', mentor: false, mentee: true, email: 'Test_email3@email.com', password_digest: 'password3'}
```

Comment:
Minor: I think we had someone to be both a mentor and a mentee as a valid scenario. How about one of these 3 users is both mentor and a mentee so all these options are covered?

Reply:
Yes that's a good point.